bucket/cuda.json

```diff
@@ -35,11 +35,12 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://developer.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/local_installers/cuda_$version_win10.exe#/dl.7z"
+                "url": "https://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/local_installers/cuda_$version_win10.exe#/dl.7z"
             }
         },
         "hash": {
-            "url": "https://developer.download.nvidia.com/compute/cuda/$majorVersion.$minorVersion/Prod/docs2/sidebar/md5sum.txt"
+            "url": "https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64&target_version=10&target_type=exelocal",
```

Comment:
What about this?

http://developer.download.nvidia.com/compute/cuda/10.1/Prod/docs/sidebar/md5sum.txt

![A](https://i.imgur.com/wFhLr0p.png)

Reply:
There is no hash for this version. And the md5 file url is changing...(docs2 ->docs)